### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by singhsterabhi**
Added styling to the popup. No links are available for the individual notices. So added a link to the Jss connect Website.
Sir, have a look.

### CODE REVIEW

**Comment by TroJan** [comment]
Why font-awesome file is added when it is being referred by CDN link?

File: popup.js [hunk 1]
```diff
@@ -1,14 +1,17 @@
 $(document).ready(function() {
-    $.get('http://210.212.85.155/', function(response){
+    $.get('http://210.212.85.155/', function(response) {
```
Comment (spyshiv) [inline popup.js]: @TroJan / @deshraj : is there any api endpoint for getting notices in the form of json or xml ? The url used in this code will return html that's temp. solution.

**Comment by spyshiv** [comment]
@singhsterabhi : there is no need of font-awesome.css if you are using cdn for that as @TroJan said. Delete that file.
`Note: you can see sources tab to know which files are served in the current DOM.`
i.e
screenshot from 2016-11-20 21 29 41

**Comment by singhsterabhi** [comment]
It was a mistake, sir @TroJan, @spyshiv . Rectified it.

**Comment by TroJan** [comment]
@singhsterabhi : thanks for your work. But this is not required, since the extension UI files are totally changed. You can help in the updated code base.

**Pull request closed without merging**